diff -Naru WTL91_5321_Final/Include/atlscrl.h WTL91_5321_Finalp/Include/atlscrl.h
--- WTL91_5321_Final/Include/atlscrl.h	2015-11-17 03:58:38.000000000 +0900
+++ WTL91_5321_Finalp/Include/atlscrl.h	2018-06-09 01:24:45.937462900 +0900
@@ -702,7 +702,7 @@
 		if(pT->AdjustScrollOffset(x, y))
 		{
 			// Children will be moved in SetScrollOffset, if needed
-			pT->ScrollWindowEx(m_ptOffset.x - x, m_ptOffset.y - y, (m_uScrollFlags & ~SCRL_SCROLLCHILDREN));
+			pT->ScrollWindowEx(-(m_ptOffset.x - x), -(m_ptOffset.y - y), (m_uScrollFlags & ~SCRL_SCROLLCHILDREN));
 			SetScrollOffset(x, y, FALSE);
 		}
 	}
